share/completions/deno.fish

```diff
@@ -1,2 +1,24 @@
 deno completions fish | source
-complete -f -c deno -n "__fish_seen_subcommand_from task" -a "(deno eval \"try {console.log(Object.keys(JSON.parse(Deno.readTextFileSync('deno.json')).tasks).join('\n'))} catch {} \")"
\ No newline at end of file
+
+# complete deno task
+set searchForDenoFilesCode '
+const denoFile = ["deno.json", "deno.jsonc", "package.json"];
+for (const file of denoFile) {
+  try {
+    Deno.statSync(file);
+    // file exists
+    if (file === "package.json") {
+      console.log(
+        Object.keys(JSON.parse(Deno.readTextFileSync(file)).scripts).join("\n"),
+      );
+    } else {
+      console.log(
+        Object.keys(JSON.parse(Deno.readTextFileSync(file)).tasks).join("\n"),
+      );
+    }
+    break;
+  } catch {
+  }
+}
+'
+complete -f -c deno -n "__fish_seen_subcommand_from task" -a "(deno eval '$searchForDenoFilesCode')"
```

Comment:
You can add `-n "__fish_is_nth_token 2"` (I think 2, but you can try 3 if that doesn't work) or you can add `-n "__fish_prev_arg_in task"` to accomplish what you want.

Reply:
thanks that works